### DISCUSSION TITLE
Sitecore XM Cloud - Component Links not editable

### DISCUSSION THREAD

**Discussion by matthewgisonno** [Sitecore/jss#1577] [Help]
### Description

I'm working on a Sitecore XM Cloud project and we're creating front-end react components in Next.js.  We're using the Sitecore Next.js JSS SDK for rendering the component data.  That said, if we have a component for a Banner, which contains a Title, Description, PrimaryCTA, and SecondaryCTA fields, the CTA's are not editable in either Pages or Experience Editor.

From both Pages and Experience Editor, I can click the Title or Description and edit the text right where it sits.  It also surrounds the field with a border to show it's actively being edited.  If I view the source, I can also see that the field is surrounded by extra markup from Sitecore to indicate the field is editable.  I do not see that same markup surrounding any links.  I will try to attach a video for reference.

I'm hoping it's a setting somewhere, but I am un-sure.  Any help would be greatly appreciated!

### Additional information

https://github.com/Sitecore/jss/assets/85267156/2450638f-7e94-4661-bb2b-e6b320aa13f3

https://github.com/Sitecore/jss/assets/85267156/756fabb0-0f83-4be5-a977-ee24c653b359

**Comment by illiakovalenko** [comment]
@matthewgisonno Hey, can you please provide more details to reproduce the issue you faced?
Can you share a Banner component implementation? In order to understand how links are rendered

**Comment by art-alexeyenko** [comment]
Closing this due as being stale for a long time.
